dracut/99switch-root/override.conf

```
@@ -0,0 +1,5 @@
+[Service]
+Type=oneshot
+EnvironmentFile=-/etc/switch-root.env
+ExecStart=
+ExecStart=systemctl --no-block switch-root /sysroot ${INIT}
```

```suggestion
ExecStart=systemctl --no-block switch-root /sysroot $INIT
```
see weird semantics here: https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines
`$INIT` if an empty string will not be passed while `${INIT}` will be passed as empty string argument

---

Oh, what a subtle difference. Funny that it worked either way. Will correct.